main.js

```diff
@@ -242,16 +242,38 @@ function saveActivities() {
 var cardsHolder = document.querySelector('.past-activities-cards-holder')
 
 function displayLoggedActivities() {
+  var color;
+
   cardsHolder.innerHTML = ``
   for (var i = 0; i < loggedActivities.length; i++) {
+    if (loggedActivities[i].category === 'Meditate') {
```

Comment:
Great job assigning a color to the cards based on category. Maybe a possible opportunity to refactor at some point with a helper function?